### PR TITLE
Update templates and Cake to use the new nugets

### PR DESCRIPTION
**Pull request by mrhelmut**
@Jjagg Here we go. Squashed remake of #6930 

Including your suggestions and added up Switch as supported by the builder package.

### CODE REVIEW

File: ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/MonoGame.Templates.CSharp.csproj [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <Title>MonoGame project templates</Title>
     <Description>A set of C# templates written for the MonoGame framework.</Description>
     <PackageId>MonoGame.Templates.CSharp</PackageId>
-    <PackageVersion>0.0.1</PackageVersion>
     <Authors>@MonoGameTeam</Authors>
+
+    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
```
Comment (Jjagg) [inline ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/MonoGame.Templates.CSharp.csproj]: @mrhelmut You missed this one. Could you make this target only netstandard2.0?

Comment (mrhelmut) [reply ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/MonoGame.Templates.CSharp.csproj]: Done!

**Comment by Jjagg** [comment]
Merged! Thank you for your effort on this!

**Comment by cra0zy** [comment]
Oh no, this PR deleted "MonoGame.Library.NetStandard.CSharp"

**Comment by cra0zy** [comment]
https://github.com/MonoGame/MonoGame/pull/6943

**Comment by mrhelmut** [comment]
My bad!

**Pull request merged**